.github/workflows/InitDev,Unit tests,Integration tests,Package.yml

```diff
@@ -0,0 +1,150 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: InitDev,Unit tests,Integration tests,Package Pull request 
+
+on:
+  pull_request:
+   branches: [main]
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+  repository_dispatch:
+    types: [triggered-from-ods-repo]
+
+env:
+  INFORMATIONAL_VERSION: "6.0"
+  BUILD_INCREMENTER: "30000"
```

Comment:
we can set to relatively near to teamcity build number ,so that continue to that .